Tests/RecoverableTests.swift

```diff
@@ -141,7 +142,7 @@ class RecoverableTests: XCTestCase {
         let exp = expectation(description: "backendServiceRecovery")
         backendService = BackendService(networkService: MockRecoverableNetworkService.protectedService(for: MockObject(title: "title", subtitle: "subtitle")), recoveryStrategy: MockFailureRecoveryStrategy())
         
-        backendService?.execute(recoverable: RecoverableRequest<MockObject>()) { result in
+        backendService?.execute(recoverable: RecoverableRequest<MockObject>()) { (result) in
```

Comment:
Same as before with the ( ). I've always used without them, but we should just make it uniform.